web/single-object.html

```diff
@@ -216,6 +219,11 @@ <h5 class="modal-title mx-auto" id="shareModalLabel">Share {{.Data.Key}}</h5>
       document.getElementById('collectingPieces').style.display = 'none';
       document.getElementById('readyForDownload').style.display = 'block';
       document.getElementById('pieces').style.display = 'block';
+
+      const fileExtension = {{.Data.Key}}.split('.').pop()
+      if (previewExtensions.includes(fileExtension)) {
+          document.getElementById('previewTag').src = `${window.location.origin}${window.location.pathname}?preview`
```

Comment:
can we just do ?wrap=0 instead of ?preview

Reply:
yes